src/components/common/Button.tsx

```diff
@@ -20,10 +20,16 @@ const Button = ({
   disabled = false,
   ...props
 }: ButtonProps) => {
+  
+  const baseClasses = 'px-4 py-2 rounded text-white transition duration-300 focus:outline-none focus:ring';
+
+  const colorClasses = disabled
+  ? 'bg-background-light cursor-not-allowed'
+  : 'bg-primary hover:bg-primary-dark focus:bg-primary-dark';
```

Comment:
This is very neat saving the classes as variables to tidy up the main component area, good idea!

Reply:
Hey Max, thanks for your feedback! I was working on improving accessibility by changing the button's colors when it's focused, hovered, or disabled to give better visual feedback. It was quite a challenging task and it took me a while to do so, especially making sure it's accessible for screen readers and keyboard users. However, I haven't tested it yet, and I need to learn how to do that.